example/fine_advection/parthenon_app_inputs.cpp

```diff
@@ -31,7 +31,15 @@ using namespace parthenon;
 // *************************************************//
 // redefine some weakly linked parthenon functions *//
 // *************************************************//
-
+namespace {
+KOKKOS_INLINE_FUNCTION
+Real sign(Real a, Real b) {
+  if (a == 0.0) {
+    return b > 0.0 ? 1.0 : -1.0;
+  }
+  return a > 0.0 ? 1.0 : -1.0;
+}
+} // namespace
```

Comment:
should this be in parthenon utils? (In another PR...)

Reply:
yeah, probably... but for later